hockey-score/src/app/shared/model/position.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum Position {
+    "defender",
```

Comment:
Maybe we will have to compare object properties or do other similar features in the future. So it may be helpful to implement enum in the following way: 
export enum Position {
    defender: "defender",
    ...
}
So later we can check the type using 'Position.defender' or use it anywhere in the code avoiding misprinting. 
What do you think?

Reply:
There is no one correct approach, it depends on how it will be used in future